tools.yml

```diff
@@ -252,3 +252,174 @@ tools:
     - id: large_job_rule
       if: 2 <= input_size
       cores: 32 # MAX
+
+## Add interactive tools
+  interactive_tool_jupyter_notebook:
+    mem: 4
+    env:
+      ## HOME dir inside the container
+      HOME: /home/jovyan
+    scheduling:
+      require:
+      - docker
+      - docker_shell_access
+  interactive_tool_pangeo_notebook:
+    mem: 4
+    env:
+      HOME: /home/jovyan
+    scheduling:
+      require:
+      - docker_shell_access
+      - docker
+  interactive_tool_climate_notebook:
+    mem: 4
+    env:
+      HOME: /home/jovyan
+    scheduling:
+      require:
+      - docker_shell_access
+      - docker
+  interactive_tool_ml_jupyter_notebook:
+    cores: 1
+    mem: 8
+    gpus: 1
+    scheduling:
+      require:
+      - docker_shell_access
+      - gpu
```

Comment:
TPV will already match `gpus` in tool rules with `gpus` on destinations, so the -gpu tag is not needed